src/Domain/Tags.hpp

```diff
@@ -55,11 +55,11 @@ namespace Tags {
 template <size_t VolumeDim, typename Frame>
```

Comment:
Does the `Domain` Tag need to be templated on `Frame` if the `Domain` class itself isn't?